### ISSUE TITLE
Startup error - non fatal

### ISSUE THREAD

**Issue by autotelik** [EnvironmentAgency/flood-risk-engine#68]
when you first run rspec or the server some errors are reported the first time a call to the controller is made .. this does not seem to affect or stop execution

```bash
Thread.exclusive is deprecated, use Mutex
/home/rubyuser/rubystack-2.2.4-0/rvm/gems/ruby-2.3.1/gems/activerecord-session_store-0.1.2/lib/active_record/session_store/session.rb:32:in `find_by_session_id'
/home/rubyuser/rubystack-2.2.4-0/rvm/gems/ruby-2.3.1/gems/activerecord-session_store-0.1.2/lib/action_dispatch/session/active_record_store.rb:66:in `block in get_session'
/home/rubyuser/rubystack-2.2.4-0/rvm/gems/ruby-2.3.1/gems/activerecord-session_store-0.1.2/lib/active_record/session_store/extension/logger_silencer.rb:45:in `silence_logger'
/home/rubyuser/rubystack-2.2.4-0/rvm/gems/ruby-2.3.1/gems/activerecord-session_store-0.1.2/lib/action_dispatch/session/active_record_store.rb:65:in `get_session'
/home/rubyuser/rubystack-2.2.4-0/rvm/gems/ruby-2.3.1/gems/rack-1.6.4/lib/rack/session/abstract/id.rb:266:in `load_session'
/home/rubyuser/rubystack-2.2.4-0/rvm/gems/ruby-2.3.1/gems/actionpack-4.2.6/lib/action_dispatch/middleware/session/abstract_store.rb:43:in `block in load_session'
/home/rubyuser/rubystack-2.2.4-0/rvm/gems/ruby-2.3.1/gems/actionpack-4.2.6/lib/action_dispatch/middleware/session/abstract_store.rb:51:in `stale_session_check!'
/home/rubyuser/rubystack-2.2.4-0/rvm/gems/ruby-2.3.1/gems/actionpack-4.2.6/lib/action_dispatch/middleware/session/abstract_store.rb:43:in `load_session'
/home/rubyuser/rubystack-2.2.4-0/rvm/gems/ruby-2.3.1/gems/actionpack-4.2.6/lib/action_dispatch/request/session.rb:180:in `load!'
/home/rubyuser/rubystack-2.2.4-0/rvm/gems/ruby-2.3.1/gems/actionpack-4.2.6/lib/action_dispatch/request/session.rb:172:in `load_for_read!'
/home/rubyuser/rubystack-2.2.4-0/rvm/gems/ruby-2.3.1/gems/actionpack-4.2.6/lib/action_dispatch/request/session.rb:89:in `[]'

...
/home/rubyuser/rubystack-2.2.4-0/rvm/gems/ruby-2.3.1/gems/puma-3.4.0/lib/puma/server.rb:271:in `block in run'
/home/rubyuser/rubystack-2.2.4-0/rvm/gems/ruby-2.3.1/gems/puma-3.4.0/lib/puma/thread_pool.rb:114:in `block in spawn_thread'

```

**Comment by woodpigeon** [comment]
https://github.com/rails/activerecord-session_store/issues/63
A deprecation warning from Ruby 2.3. Fixed in activerecord-session_store 1.0.0.pre so as long as we update once 1.0.x is released this will disappear.

**Comment by woodpigeon** [comment]
Fixed in a PRD to upgrade ar session_store
